Casks/zeplin.rb

```diff
@@ -1,5 +1,5 @@
 cask "zeplin" do
-  version "3.21.1,1211"
+  version "3.22"
```

Comment:
@DSGND The version number will need to have the build number included to pass CI

```suggestion
  version "3.22,1266"
```